assets/js/base/context/providers/store-notices/components/store-notices-container.js

```diff
@@ -50,7 +51,7 @@ const StoreNoticesContainer = ( { className, notices, removeNotice } ) => {
 						}
 					} }
 				>
-					{ props.content }
+					{ decodeEntities( props.content ) }
```

Comment:
Hey @manospsyx - I discussed briefly with @xristos3490 and he suggested using `decodeEntities` in one place (here - the store notices container) rather than everywhere else we call `createNotice`

What do you think?

I tested it locally and it works as expected (with `decodeEntities` in `assets/js/base/utils/errors.js` removed). If you agree, we can revert `errors.js` and just keep the change in the container.

Reply:
I remember there's at least one instance where `decodeEntities` is used outside -- I wasn't sure if that's intentional. I agree with Chris, but let's make sure that we're not double-decoding in other places. Can you check usage of `decodeEntities` in other locations to make sure?

Reply:
As discussed with @senadir over slack

> My personal preference is to not call something unless needed seeing that you can't opt out of it in a clean way, I think decoding all server returned stings makes more sense than decoding at the createNotice level.